doc/radosgw/s3select.rst

```diff
@@ -32,30 +32,29 @@ the better the performance of s3select**.
 Basic Workflow
 --------------
     
-S3-select query is sent to RGW via `AWS-CLI
+S3 Select queries are sent to RGW via `AWS-CLI
 <https://docs.aws.amazon.com/cli/latest/reference/s3api/select-object-content.html>`_
 
-It passes the authentication and permission process as an incoming message
-(POST). **RGWSelectObj_ObjStore_S3::send_response_data** is the “entry point”,
-it handles each fetched chunk according to input object-key.
-**send_response_data** is first handling the input query, it extracts the query
-and other CLI parameters.
+S3 Select passes the authentication and permission process as an incoming
+message (POST). ``RGWSelectObj_ObjStore_S3::send_response_data`` is the entry
+point and handles each fetched chunk according to the object key that was input.
+``send_response_data`` is the first to handle the input query: it extracts the
+query and other CLI parameters.
    
-Per each new fetched chunk (~4m), RGW executes an s3-select query on it. The
-current implementation supports CSV objects and since chunks are randomly
-“cutting” the CSV rows in the middle, those broken-lines (first or last per
-chunk) are skipped while processing the query.   Those “broken” lines are
-stored and later merged with the next broken-line (belong to the next chunk),
-and finally processed.
-   
-Per each processed chunk an output message is formatted according to `AWS
+RGW executes an S3 Select query on each new fetched chunk (~4m). The current
```

Comment:
Maybe ~4m should be something like "up to 4 MB"?

Reply:
Accepted.